src/main/resources/META-INF/plugin.xml

```diff
@@ -23,6 +23,8 @@ limitations under the License.
     <!-- please see https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
          on how to target different products -->
     <depends>com.intellij.modules.platform</depends>
+    <depends>com.intellij.modules.vcs</depends>
+    <depends>Git4Idea</depends>
```

Comment:
remove?

Reply:
The featureHistory in HAnS-Viz can't be fetched if I delete this